atlasdb-config/build.gradle

```diff
@@ -1,32 +1,41 @@
 apply from: "../gradle/publish-jars.gradle"
 apply plugin: 'org.inferred.processors'
+apply plugin: 'com.github.johnrengelman.shadow'
 
 apply from: "../gradle/shared.gradle"
 
+configurations {
+    explicitShadow
+    compile.extendsFrom(explicitShadow)
+    shadow.extendsFrom(explicitShadow)
```

Comment:
Yep. Unfortunately I think I have to maintain this decision, to avoid making the ALPN java-agent a compile dep.